locale/it-IT/multipletab/multipletab.properties

```diff
@@ -6,7 +6,7 @@ moveTabsToGroup.newGroup.message=%S tab(s) are moved to a new group. Please give
 compatibility_TMP_warning_title=Conflitto tra Tab Mix Plus e Multiple Tab Handler
 compatibility_TMP_warning_text=Multiple Tab Handler fornisce la possibilità di selezionare le schede attraverso Ctrl+clic o Maiusc+clic. Dal momento che anche Tab Mix Plus utilizza tali combinazioni, il loro utilizzo attiverà entrambe le funzionalità. Selezionare quale mantenere attiva.
 compatibility_TMP_warning_text_mac=Multiple Tab Handler fornisce la possibilità di selezionare le schede attraverso ⌘+clic o Maiusc+clic. Dal momento che anche Tab Mix Plus utilizza tali combinazioni, il loro utilizzo attiverà entrambe le funzionalità. Selezionare quale mantenere attiva.
-compatibility_TMP_warning_text_note=(*È possibile modificare tali impostazioni in un secondo momento agendo nella finestra di configurazione di ciascuna estensione.)
+compatibility_TMP_warning_text_note=(*È possible modificare tali impostazioni in un secondo momento agendo nella finestra di configurazione di ciascuna estensione.)

```

Comment:
Same to above.  I cannot determine it because I cannot read any Italian sentence.